downstream/modules/platform/ref-get-started-credential-types.adoc

```diff
@@ -2,17 +2,20 @@
 
 = Getting started with credential types
 //[ddacosta] Consider rewriting this as a procedure.
-From the navigation panel, select {MenuAMCredentialType}.
-If no custom credential types have been created, the *Credential Types* prompts you to add one.
 
+.Procedure
+. From the navigation panel, select {MenuAECredentials}.
+If no custom credential types have been created, the *Credential Types* page prompts you to add one.
++
 //image:credential-types-home-empty.png[Credential Types - empty]
-
++
 If credential types have been created, this page displays a list of existing and available Credential Types.
-
++
 //image:credential-types-home-with-example-types.png[Credential Types - example credential types]
 
-To view more information about a credential type, click the name of a credential or the Edit image:leftpencil.png[Edit, 15,15] icon.
+. Select the name of a credential or the Edit image:leftpencil.png[Edit, 15,15] icon to view more information about a credential type, .
 
-Each credential type displays its own unique configurations in the *Input Configuration* field and the *Injector Configuration* field, if
-applicable.
+. On the *Details* tab, each credential type displays its own unique configurations in the *Input Configuration* field and the *Injector Configuration* field, if applicable.
 Both YAML and JSON formats are supported in the configuration fields.
+
+//NOTE The Back to Credential Types Tab throws an error.
```

Comment:
FYI: This is a known issue currently being worked on with Jira ticket https://issues.redhat.com/browse/AAP-27018.